### DISCUSSION TITLE
Rate组件设置两种icon

### DISCUSSION THREAD

**Discussion by Codyooo** [ant-design/ant-design#28267] [Q&A] [answered]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
评分组件的选中状态和非选中状态可能是两种不同的图标

### What does the proposed API look like?
```
<Rate 
   character={<Svg1 />}
   emptyCharacter={<Svg2 />}
/>
```

**Comment by xrkffgg** [comment]
`character`   `(RateProps) => ReactNode`  应该满足
https://ant.design/components/rate-cn/#components-rate-demo-character-function

**Comment by afc163** [comment, marked as answer]
https://codesandbox.io/s/zidingyizifu-antd492-forked-dwqzr?file=/index.js
